backend/services/team_service.py

```diff
@@ -323,14 +327,11 @@ def get_team_as_dto(team_id: int, user_id: int, abbreviated: bool) -> TeamDTO:
 
             team_dto.team_projects.append(project_team_dto)
 
-        org_projects = OrganisationService.get_projects_by_organisation_id(
-            team.organisation.id
-        )
-        for org_project in org_projects:
-            org_project_dto = OrganisationProjectsDTO()
-            org_project_dto.project_id = org_project.id
-            org_project_dto.project_name = org_project.name
-            team_dto.organisation_projects.append(org_project_dto)
+        org_project_dto = OrganisationProjectsDTO()
```

Comment:
I don't understand the need for organisation projects here - also why do we have organisation ID assigned to project ID field?

Reply:
I did the verification and proceeded to delete these lines